packages/asc-ui/src/theme/CompactThemeProvider.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react'
+import { storiesOf } from '@storybook/react'
+import Paragraph from '../components/Paragraph'
+import Heading from '../components/Heading'
+import CompactThemeProvider from './CompactThemeProvider'
+
+storiesOf('Atoms/Typography/CompactThemeProvider', module).add(
+  'without theme ',
+  () => (
+    <>
+      <div>
+        <Heading $as="h3">
+          I don&apos;t have the CompactThemeProvider applyed.
+        </Heading>
+        <Paragraph>
+          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Debitis
+          deleniti dicta doloremque eveniet explicabo id ipsum omnis placeat
+          rerum suscipit.
+        </Paragraph>
+      </div>
+      <div>
+        <CompactThemeProvider>
```

Comment:
This is already in the Paragraph story isn't it?

Reply:
will delete the whole story then